schema.sql

```diff
@@ -0,0 +1,175 @@
+                                                        `category_id` INT NOT NULL AUTO_INCREMENT,
+                                                        `category_title` VARCHAR(100) NOT NULL,
+                                                        `symbolic_code` VARCHAR(100) NOT NULL,
+                                                        PRIMARY KEY (`category_id`),
+                                                        UNIQUE INDEX `title_UNIQUE` (`category_title` ASC) VISIBLE)
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`item`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`item` (
+                                                    `item_id` INT NOT NULL AUTO_INCREMENT,
+                                                    `creation_date` TIMESTAMP(14) NOT NULL,
+                                                    `item_title` VARCHAR(100) NOT NULL,
+                                                    `description` VARCHAR(255) NOT NULL,
+                                                    `image` VARCHAR(255) NOT NULL,
+                                                    `start_price` INT NOT NULL,
+                                                    `completion_date` TIMESTAMP(14) NOT NULL,
+                                                    `bet_step` INT NOT NULL,
+                                                    PRIMARY KEY (`item_id`))
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`bet`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`bet` (
+                                                   `bet_id` INT NOT NULL AUTO_INCREMENT,
+                                                   `bet_date` TIMESTAMP(14) NOT NULL,
+                                                   `total` DECIMAL(10,2) NOT NULL,
+                                                   PRIMARY KEY (`bet_id`))
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`user`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`user` (
+                                                    `iser_id` INT NOT NULL AUTO_INCREMENT,
+                                                    `registration_date` TIMESTAMP(14) NOT NULL,
+                                                    `email` VARCHAR(100) NOT NULL,
+                                                    `name` VARCHAR(45) NOT NULL,
+                                                    `password` VARCHAR(45) NOT NULL,
+                                                    `contacts` VARCHAR(255) NOT NULL,
+                                                    PRIMARY KEY (`iser_id`),
+                                                    UNIQUE INDEX `email_UNIQUE` (`email` ASC) VISIBLE)
```

Comment:
[ERROR in query 10] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'VISIBLE,
                                                           INDEX `item_' at line 9